src/styles/_card_product.scss

```diff
@@ -0,0 +1,136 @@
+.card {
+  font-family: Roboto, serif;
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding-inline: 16px;
+  padding-top: 32px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  .image{
+    width: 160px;
+    height: 134px;
+  }
+
+  &__description-product {
+    width: 166px;
+    padding-top: 40px;
+    .product-title {
+
+
+      .product-code {
+        color: $text;
+        padding-top: 4px;
+        font-size: 10px;
+        font-weight: 400px;
+        line-height: 14px;
+      }
+    }
+  }
+
+  .title {
+      color: $title;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+
+
+    }
+
+  .product-grade {
+    position: relative;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    padding-top: 16px;
+    .stars {
+      display: flex;
+      gap: 4px;
+    }
+
+    .stars_star {
+      height: 16px;
+      width: 16px;
+      background-repeat: no-repeat;
+      background-position: center;
+    }
+
+    .stars .stars_star:not(:nth-child(-n+4)) {
+      background-image: url(../images/star.svg);
+    }
+
+    .stars .stars_star:nth-child(-n+4) {
+      background-image: url(../images/star-active.svg);
+    }
+
+  }
```

Comment:
you need to add all stars style from previous task for have possibility to change quantity of active stars.
also need to move this like separate block to other scss file